emacs/inferior-lfe.el

```diff
@@ -129,8 +136,8 @@ Customization: Entry to this mode runs the hooks on `comint-mode-hook' and
 (defalias 'run-lfe 'inferior-lfe)
 
 (defun lfe-eval-region (start end &optional and-go)
-  "Send the current region to the inferior LFE process.
-Prefix argument means switch to the LFE buffer afterwards."
+  "Send the current region (from `START' to `END') to the inferior LFE process.
+`AND-GO' means switch to the LFE buffer afterwards."
```

Comment:
I don't love this, but Flycheck is satisfied that `AND-GO` is present in the docstring.